src/types.ts

```diff
@@ -190,6 +190,13 @@ export type CellKeyDownArgs<TRow, TSummaryRow = unknown> =
   | SelectCellKeyDownArgs<TRow, TSummaryRow>
   | EditCellKeyDownArgs<TRow, TSummaryRow>;
 
+export interface CellSelectArgs<TRow, TSummaryRow = unknown> {
+  idx: number;
+  rowIdx: number;
+  row: TRow;
+  column: CalculatedColumn<TRow, TSummaryRow>;
```

Comment:
```suggestion
  rowIdx: number;
  row: TRow;
  column: CalculatedColumn<TRow, TSummaryRow>;
```
We can easily get `idx` from `column`

Reply:
Removed in b7037d1 and 4612a95.